doc/filters/batch.rst

```diff
@@ -40,3 +40,9 @@ The above example will be rendered as:
             <td>No item</td>
         </tr>
     </table>
+
+Arguments
+---------
+
+* ``batch size``: Count in one batch. Fractional number will be rounded up.
```

Comment:
They keys here must be the ones you can use in a template, so `size` and `fill`. I'm going to fix that after merging.